model/navigation/LtiNavigationService.php

```diff
@@ -127,23 +136,28 @@ protected function shouldShowThankYou(LtiLaunchData $launchData): bool
         return $this->getOption(self::OPTION_THANK_YOU_SCREEN);
     }
 
-    /**
-     * @return string
-     */
     protected function buildThankYouUrl(): string
     {
-        return $this->getServiceLocator()->get(UrlHelper::class)->buildUrl(
-            'thankYou',
+        return $this->getUrlHelper()->buildUrl('thankYou', 'DeliveryRunner', 'ltiDeliveryProvider');
```

Comment:
- [ ] Lets rever unnecessary refactoring to reduce noise on PR review, please. Thanks!